### ISSUE TITLE
tsc really slow

### ISSUE THREAD

**Issue by maku** [NativeScript/sample-Tasks#32]
I tried the typescript compiler (tsc) and the compiling is really slow (e.g. 20s). I guess this is because of the type definitions. Is the time improvable? In a real world dev scenario it would be a no go.

**Comment by x4080** [comment]
I agree, even tsc -w -p app is not updating maybe a minute or so, so I gave up :)
